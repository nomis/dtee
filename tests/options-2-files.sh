#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

run_test \
	"-o" "stdout_append1" \
	"-e" "stderr_append1" \
	"-c" "combined_append1" \
	"-O" "stdout_overwrite1" \
	"-E" "stderr_overwrite1" \
	"-C" "combined_overwrite1" \
	\
	"-o" "stdout_append2" \
	"-e" "stderr_append2" \
	"-c" "combined_append2" \
	"-O" "stdout_overwrite2" \
	"-E" "stderr_overwrite2" \
	"-C" "combined_overwrite2" \
	"--debug-options"
RET=$?

echo RET $RET
if [ $RET -eq 0 ]; then
	exit 0
fi
exit 1
