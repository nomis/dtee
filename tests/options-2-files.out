combined-append=combined_append1
combined-append=combined_append2
combined-overwrite=combined_overwrite1
combined-overwrite=combined_overwrite2
debug-options=1
err-append=stderr_append1
err-append=stderr_append2
err-overwrite=stderr_overwrite1
err-overwrite=stderr_overwrite2
out-append=stdout_append1
out-append=stdout_append2
out-overwrite=stdout_overwrite1
out-overwrite=stdout_overwrite2
